### PR TITLE
Fix remaining lints

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,11 +1,12 @@
 module github.com/majewsky/schwift
 
-go 1.20
+go 1.21
```
Comment (majewsky) [inline go.mod:3]: Please revert. I set this deliberately in https://github.com/majewsky/schwift/commit/17c1ae81f02a4b2a24e79d4018dfb1061962420d yesterday.

File: object.go [hunk 2]
```diff
@@ -456,6 +461,7 @@ func (o *Object) Download(opts *RequestOptions) DownloadedObject {
 	}.Do(o.c.a.backend)
 	var body io.ReadCloser
 	if err == nil {
+		defer resp.Body.Close()
```
Comment (majewsky) [inline object.go:464]: This is incorrect. This function moves `resp.Body` into the returned DownloadedObject.

**Review by majewsky — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (SuperSandro2000) [reply go.mod:3]: Then we can't use go-bits and errext.As

Comment (majewsky) [reply go.mod:3]: Then we should copy the respective functions over into this package as `internal`. It's only two small functions, should not be too much duplication.

**Review by majewsky — APPROVED** [review]

**Pull request merged**